storage/storage.go

```diff
@@ -2001,6 +2001,19 @@ func setEncryptionHeaders(headers http.Header, key []byte, copySource bool) erro
 	return nil
 }
 
+// toProtoCommonObjectRequestParams sets client-side encryption to the proto library's CommonObjectRequestParams.
+func toProtoCommonObjectRequestParams(key []byte) *storagepb.CommonObjectRequestParams {
+	if key == nil {
+		return nil
+	}
+	keyHash := sha256.Sum256(key)
+	return &storagepb.CommonObjectRequestParams{
+		EncryptionAlgorithm:      "AES256",
```

Comment:
Nit: Does it make sense to refactor `"AES256"` into an unexported, global constant since it is used in two places? e.g. `const aes256Algorithm = "AES256"`. Not too opinionated on this though.

Reply:
I think that sounds reasonable. Updated to make it a const.